src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -725,9 +723,18 @@ public class SimpleEvents {
 				.description("Called when a player drags an item in their cursor across the inventory.")
 				.examples(
 						"on inventory drag:",
-						"\tif player's current inventory is {_gui}:",
-						"\t\tsend \"You can't drag your items here!\" to player",
-						"\t\tcancel event"
+							"\tif player's current inventory is {_gui}:",
```

Comment:
`{_gui}` is never initialized in this event, so global var would be more correct here.
```suggestion
							"\tif player's current inventory is {inventories::custom-gui}:",
```